source/microbit/help.c

```diff
@@ -54,7 +54,8 @@ STATIC const char *help_text =
 "  CTRL-E        -- enter paste mode, turning off auto-indent\n"
 "\n"
 "Available modules: antigravity, array, collections, gc, love, math,\n"
-"micropython, music, neopixel, random, struct, sys, this\n"
+"micropython, music, neopixel, os, radio, random,\n" 
+"speech, struct, sys, this,\n"
```

Comment:
Sorry to be a pain, but can you please make these 2 new lines into 1 line, with all the modules listed from "micropython" to "this"?  They will fit within 80 chars and it's best to reduce the total number of lines.  Also, the end of the line should be left without a comma or period, just a newline like it was previously.  Thanks!